mapper/Module.md

```diff
@@ -0,0 +1,6 @@
+Prints "Hello, World!" with a main function.
```

Comment:
Let's add a proper explanation to the module

Reply:
Done